frontend/src/components/AddPage.tsx

```diff
@@ -6,8 +6,8 @@ type Props = {
     onAddGif: (data: GifWithoutId) => void;
 }
 
+
 export default function AddPage(props: Props){
-    const newGif: GifWithoutId = {name: "", description: "", price: ""}
 
-    return <InputForm onSubmitGif={props.onAddGif} gif={newGif} legend={"Add new Gift"} backUrl="/" placeholder=""/>
+    return <InputForm onAddGif={props.onAddGif} legend={"Add new Gift"} backUrl="/" placeholder="" />
 }
```

Comment:
Newline vergessen? ;)
```suggestion
}

```